oaknational/Oak-Web-Application
# Fix/lesq 283/keywords typo

## Description 

- Fix the typo in keywords component
- update related tests to look for correct spelling
- I had issues with tests timing out and failing, this wasn't happening consistently however :( I was getting different results when running `npm run test:ci` and when the pre-push hook was running the same script. I updated the failing tests to remove the `waitFor` wrapping the assertions as it didn't seem like there was anything that would need awaiting anyway, and this fixed it.
  
Fixes #. 
[LESQ-283](https://www.notion.so/oaknationalacademy/Typo-in-lesson-details-Key-words-9a1c15d81ba44d31a4b1584e316442a5?pvs=4)

## How to test

1. Go to https://deploy-preview-1884--oak-web-application.netlify.thenational.academy
2. Navigate to a lesson
3. You should see `Keywords` instead of `Key Words` under Lesson Details


## Checklist
